ton_client/src/debot/dengine.rs

```diff
@@ -179,6 +182,51 @@ impl DEngine {
         }
     }
 
+    pub async fn send(&mut self, source: String, func_id: u32, params: String) -> ClientResult<()> {
+        debug!("send from {} id = {} params = {}", source, func_id, params);
+        let params = serde_json::from_str(&params)
+            .map_err(|e| Error::invalid_json_params(e) )?;
+        let abi = Contract::load(self.raw_abi.as_bytes()).unwrap();
```

Comment:
No `unwrap`s are allowed in SDK. Check the result

Reply:
Fixed